cheerio-scraper/src/crawler_setup.js

```diff
@@ -210,7 +215,11 @@ class CrawlerSetup {
             ),
             pageFunctionArguments: {
                 $,
-                html,
+                get html() {
+                    log.warning('Cheerio Scraper: Parameter context.html is deprecated use context.body instead.');
```

Comment:
There's a `log.deprecated` function. Does the same as `log.warning` but only logs it once.